pages/ReportsVerification.tsx

```
@@ -224,7 +206,91 @@ export default function ReportsVerification() {
+                    reason: r.reason,
+                    dateReporting: formatDateLocal(r.date_reporting),
+                    state: mapApiStateToUi(r.state),
+                    reporterName: r.reporter_name,
+                    reportedName: r.reported_name,
+                })));
+                setStats(apiStats);
+            } catch (e: any) {
+                if (!mounted) return;
+                setErrorMsg(e?.message ?? t('loadReportsError'));
+            } finally {
+                if (!mounted) return;
```

Biome (2.3.13)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)